core/templates/dev/head/pages/skill_editor/editor_tab/SkillConceptCardEditorDirective.js

```diff
@@ -48,6 +48,15 @@ oppia.directive('skillConceptCardEditor', [
             return true;
           };
 
+          $scope.$on(EVENT_SKILL_REINITIALIZED, function() {
+            $scope.bindableFieldsDict = {
```

Comment:
Perhaps extract this and lines 39-42 into var initBindableFieldsDict -- try not to repeat code.

Reply:
Done.